clang/lib/StaticAnalyzer/Checkers/STLAlgorithmModeling.cpp

```diff
@@ -97,11 +99,12 @@ bool STLAlgorithmModeling::evalCall(const CallEvent &Call,
   if (!Handler)
     return false;
 
-  return (this->**Handler)(C, CE);
+  return (this->**Handler)(C, CE, Call.getCFGElementRef());
```

Comment:
Ideally, here we would pass the `Call` instead of `CE`. Then the handlers could dig up the Elem if they need that using the `Call.getCFGElementRef()`. I'd prefer this over what we propose here right now unless there is some compelling argument doing it this way.